### PR TITLE
DROP requirements and test fixes

### PR DESCRIPTION
**Pull request by mumichae**
* Constrain FRASER>=1.2.0

* Use pytest

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mumichae** [comment]
@BiocondaBot rebuild

**Comment by mumichae** [comment]
@bioconda/core
Hi, `mulled-build` fails when using pytest for drop. I'm not sure whether it's because of the way I handle pytest dependencies or whether it can be fixed by a simple rebuild.

File: recipes/drop/meta.yaml [hunk 1]
```diff
@@ -56,24 +56,24 @@ requirements:
...
   commands:
-    - drop --help
-    - mkdir -p /tmp/drop_test
-    - cd /tmp/drop_test
-    - drop demo > /dev/null
-    - snakemake -n > /dev/null
-    - snakemake --cores 1 --jobs 1
-    - cd
-    - rm -rf /tmp/drop_test
+    - pip install -r tests/requirements.txt
```
Comment (dpryan79) [inline recipes/drop/meta.yaml:69]: Please just list the additional requirements in the `requires:` section

**Comment by dpryan79** [comment]
When the mulled test is run, it only has access to a minimal system with drop and its dependencies. So there won't be a `tests/requirements.txt` file for pip to read. A few comments:

- If possible, put the test requirements in the `requires:` section
- If you do need files that aren't installed as part of the package, then move this test from meta.yaml to run_test.sh. The commands in run_test.sh are skipped in the mulled test, so they can make use of uninstalled files that are present in the source directory.

**Comment by mumichae** [comment]
Thanks for the quick reply!

I moved the test commands from the `meta.yaml` to the `run_test.sh` script. I'm assuming that `run_test.sh` was run automatically, however I couldn't quite follow that in the circle-ci output. Where is the test run on circleci?

**Comment by mumichae** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**